src/runtime/Runtime.hpp

```diff
@@ -8,8 +8,8 @@
  * environments.
  */
 #pragma once
-#include <stddef.h> // NOLINT(modernize-deprecated-headers)
-#include <stdint.h> // NOLINT(modernize-deprecated-headers)
+#include <stddef.h>
+#include <stdint.h>
```

Comment:
:warning: **modernize-deprecated-headers** :warning:
inclusion of deprecated C++ header 'stdint.h'; consider using 'cstdint' instead
```suggestion
#include <cstdint>```